requests/src/main/scala/uk/ac/wellcome/platform/api/requests/RequestsApi.scala

```diff
@@ -27,22 +22,16 @@ trait RequestsApi extends CustomDirectives {
       post {
         entity(as[ItemRequest]) {
           itemRequest: ItemRequest =>
-            val canonicalId =
-              CanonicalId(itemRequest.itemId)
-
-            val result = stacksWorkService.requestHoldOnItem(
-              userIdentifier = userIdentifier,
-              itemId = canonicalId,
-              neededBy = None
-            )
-
-            val accepted = (StatusCodes.Accepted, HttpEntity.Empty)
-            val conflict = (StatusCodes.Conflict, HttpEntity.Empty)
-
-            onComplete(result) {
-              case Success(HoldAccepted(_)) => complete(accepted)
-              case Success(HoldRejected(_)) => complete(conflict)
-              case Failure(err)             => failWith(err)
+            // TODO: We get the work ID as part of the item request, although right now
+            // it's only for future-proofing, in case it's useful later.
+            // Should we query based on the work ID?
```

Comment:
I think we probably should do this, but it's yet another bit of refactoring, and has some rough edges.

I'd rather get this patch merged (which replicates how we used to do item lookups), and then come back and tackle this in a separate PR.